ssl/quic/quic_impl.c

```diff
@@ -2177,6 +2177,53 @@ struct quic_write_again_args {
     int                 err;
 };
 
+#define MAX_WRITE_BUF_SIZE      (6 * 1024 * 1024)
```

Comment:
Since this seems like an arbitrary number, can you add some comment to it, including the 500ms/100 Mb/s.